frontend/resourcebundles/src/main/resources/GitMacheteBundle.properties

```diff
@@ -64,6 +64,7 @@ action.GitMachete.BaseOverrideForkPointAction.dialog.override-fork-point.label.H
 action.GitMachete.BaseOverrideForkPointAction.dialog.override-fork-point.title=Override Fork Point
 action.GitMachete.BaseOverrideForkPointAction.dialog.override-fork-point.radio-button.parent=Parent branch ({0})
 action.GitMachete.BaseOverrideForkPointAction.dialog.override-fork-point.radio-button.inferred=Inferred commit ({0})
+action.GitMachete.BaseOverrideForkPointAction.dialog.override-fork-point.radio-button.custom=Manually picked commit:
```

Comment:
Also, I'd automatically switch radio to `custom` if a choice is made within the combobox... as for now, one can end up making a choice in combobox, but the radio button remains unchanged

Reply:
I can think of a formerly existing place in IntelliJ which such the same UX glitch... coincidentally, also related to _overriding_ things 😅 

![image](https://user-images.githubusercontent.com/3383210/196207985-223bd438-a42a-497a-b936-0b0a1840f368.png)

Fixed in 2022.1 or 2022.2, but in the past you could select association in combobox, and nothing happened ☹️

Reply:
done! 

<img width="765" alt="image" src="https://user-images.githubusercontent.com/19799111/196255496-de52d946-3bdc-445b-adf4-3cc1bc77a57b.png">